src/controllers/cohort.js

```diff
@@ -10,3 +10,12 @@ export const create = async (req, res) => {
     return sendMessageResponse(res, 500, 'Unable to create cohort')
   }
 }
+
+export const getCohort = async (req, res) => {
```

Comment:
Very minor, but might be worth calling this method `getAll` - `getCohort` sounds like gets a single cohort on first readinf, but also renaming to `getAll` matches the convention from the user controller.